Changed ASCONF support according to IETF-124 discussion

teiclap opened the pull request.

teiclap committed:
Changed ASCONF support according to IETF-124 discussion

teiclap commented on draft-ietf-tsvwg-sctp-dtls-chunk.md:
```diff
@@ -356,19 +356,11 @@ in {{RFC9260}}.
+Support for Dynamic Address Reconfiguration {{RFC5061}} in an SCTP
+association using DTLS Chunk is limited to the case where ASCONF
+is sent from an IP address already known.
+SCTP packets coming from unknown IP addresses will be silently
```
That's fine with me. I just removed the whole sentence.

teiclap committed:
Removed redundant sentence as proposed by Michael

gloinul committed:
Update draft-ietf-tsvwg-sctp-dtls-chunk.md

We need the sentence making clear that ASCONF can be sent when used on a path already bound to particular association.

gloinul commented:
Made a change to retain the make before break support that exist. But, this only addresses half the issue, we need to do something about the SCTP-AUTH aspect. So lets agree on direction there and add this to this PR also.

gloinul commented:
The proposal at IETF-124 meeting slides: https://datatracker.ietf.org/meeting/124/materials/slides-124-tsvwg-dtls-chunk-00
Was to rely on DTLS chunk for authentication. So we need to add the Updates of RFC 5061 text.

teiclap committed:
Inserted considerations about authentication

teiclap committed:
Merge branch 'issue28' of https://github.com/gloinul/draft-westerlund-tsvwg-sctp-dtls-chunk into issue28

gloinul committed:
Update draft-ietf-tsvwg-sctp-dtls-chunk.md

teiclap committed:
Updated with Magnus suggestions

teiclap committed:
Merge branch 'issue28' of https://github.com/gloinul/draft-westerlund-tsvwg-sctp-dtls-chunk into issue28

teiclap committed:
Trailing space...

teiclap committed:
non-compatible

gloinul committed:
Update draft-ietf-tsvwg-sctp-dtls-chunk.md

Further word smithing and adding to abstract and introduction regarding the updates.

teiclap committed:
Trailing spaces...

tuexen committed:
Merge branch 'main' into issue28
